scripts/openshiftci-periodic-tests.sh

```diff
@@ -40,6 +40,6 @@ if [ $error ]; then
     exit -1
 fi
 
-cp -r reports $ARTIFACT_DIR 
+cp -r reports tests/reports $ARTIFACT_DIR 
```

Comment:
This will copy both `reports` and `tests/reports` into `$ARTIFACT_DIR` and merge the contents of two directories. Is that what we want?
```sh
# a, b and c are directories; c is empty so far.
$ tree a b c
a
└── hello
b
└── a
    └── hi
c

1 directory, 2 files

$ cp -r a b/a c

$ tree c    
c
└── a
    ├── hello
    └── hi
```

Reply:
yes

Reply:
Ack, thanks. Only wanted to confirm it.